models/webhook.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Gogs Authors. All rights reserved.
+// Copyright 2017 Gitea. All rights reserved.
```

Comment:
Only Gitea copyright is in files that are new and was not in Gogs. In places where new things are added new copyright line is added, see:
https://github.com/go-gitea/gitea/blob/a2019775903902622dc80c2231f105e3213f8a60/main.go#L2
Also it should be `The Gitea Authors`